marked-element.html

```diff
@@ -75,7 +75,7 @@
         display: none !important;
```

Comment:
Sorry, I must be missing something. How would setting a display style on :host override it?